bump-angular-version-to-19

akhromets opened the pull request.

dorpeled commented:
Would help my team :)
Thanks!

Waschndolos commented:
Same here. Would be great to use it on Angular 19. Thanks for the effort!

ghbob commented:
@RobbieTheWagner sorry for the ping.
Could you have a look at this Angular 19 PR?
Thank you!

bastienmoulia commented:
> This seems fine, but it seems like tests are not running for some reason 🤔

https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/approving-workflow-runs-from-public-forks

RobbieTheWagner commented:
@bastienmoulia there is usually something asking me to approve though, and I do not see that here.

ghbob commented:
Maybe an issue with the new merge box?

Old: `https://github.com/shipshapecode/angular-shepherd/pull/2566?new_mergebox=false`
New: `https://github.com/shipshapecode/angular-shepherd/pull/2566?new_mergebox=true`

old one says
> 1 workflow awaiting approval

RobbieTheWagner commented:
@ghbob ah, yes, thank you! @akhromets it looks like tests are failing

akhromets commented:
Hi @RobbieTheWagner , I'll have a look, thanks!

akhromets commented:
Hi @RobbieTheWagner , I’ve updated the tests.
Since components are standalone by default in Angular v19, I’ve explicitly marked them as standalone in the component decorators to avoid any confusion. I also replaced declarations with imports in the `TestBed.configureTestingModule` section.

RobbieTheWagner commented:
Thanks @akhromets!

ghbob commented:
@RobbieTheWagner will this be released as v19? thanks.

RobbieTheWagner commented:
@ghbob yes, this has now been released! I also updated Shepherd to 14.x which required messing with some TS stuff, so let me know if you all run into any issues.

ghbob commented:
Thank you!

In case anyone else didn't see this:
With Shepherd 14+ the license has changed: <https://github.com/shipshapecode/shepherd?tab=License-1-ov-file>

RobbieTheWagner commented:
Thanks for the callout @ghbob. We did update the license, but you only need to pay if you are running a commercial product, so hopefully it is not too restrictive for folks.
